src/swift/Block.swift

```diff
@@ -104,8 +104,8 @@ public class DispatchWorkItem {
 /// uses @convention(block). This is to avoid unnecessary bridging between
 /// C blocks and Swift closures, which interferes with dispatch APIs that depend
 /// on the referential identity of a block. Particularly, dispatch_block_create.
-internal typealias _DispatchBlock = @convention(block) () -> Void
-internal typealias dispatch_block_t = @convention(block) () -> Void
+internal typealias _DispatchBlock = @escaping @convention(block) () -> Void
+internal typealias dispatch_block_t = @escaping @convention(block) () -> Void
```

Comment:
my point is that I find it troubling to annotate the type, as if the overlay has to do a
```
dispatch_sync(q) { ... }
```

some day for some weird reason, we will get a poorer performance for it. The importer should already have annotated the C functions (that are properly `__attribute__((noescape))` where it makes sense) properly already anyway, so I think the one on dispatch_block_t is wrong.

@mwwa  do I make sense?